mlflow/pipelines/utils/tracking.py

```diff
@@ -193,10 +193,19 @@ def apply_pipeline_tracking_config(tracking_config: TrackingConfig):
                 "Experiment with name '%s' does not exist. Creating a new experiment.",
                 tracking_config.experiment_name,
             )
-            client.create_experiment(
-                name=tracking_config.experiment_name,
-                artifact_location=tracking_config.artifact_location,
-            )
+            try:
+                client.create_experiment(
+                    name=tracking_config.experiment_name,
+                    artifact_location=tracking_config.artifact_location,
+                )
+            except RestException:
+                # Inform user they should create an experiment and specify it in the pipeline
+                # config if an experiment with the pipeline name can't be created.
+                raise MlflowException(
+                    f"Could not create an experiment with name {tracking_config.experiment_name}."
+                    f"Please create an experiment for this pipeline and specify its name in your"
```

Comment:
Can we tell the user specifically what YAML config key they should use for specifying this config?

Reply:
And that the user that is running the pipeline should have write access to the experiment path?

Reply:
Will do @dbczumar. I think that is implied @mahesh-venkatachalam; they would face that error message upon trying to create an experiment in a place they didn't have access to.